tests/validation/validators/slackChannelName.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { slackChannelName } from 'sefirot/validation/validators'
+
+describe('validation/validators/slackChannelName', () => {
+  test('it validates whether the value is valid', () => {
+    expect(slackChannelName('')).toBe(true)
+    expect(slackChannelName('1'.repeat(80))).toBe(true)
+
+    expect(slackChannelName('1'.repeat(81))).toBe(false)
+    expect(slackChannelName(' ')).toBe(false)
+    expect(slackChannelName(',')).toBe(false)
+    expect(slackChannelName('.')).toBe(false)
+    expect(slackChannelName('!')).toBe(false)
+    expect(slackChannelName('@')).toBe(false)
+    expect(slackChannelName('#')).toBe(false)
+    expect(slackChannelName('$')).toBe(false)
+    expect(slackChannelName('%')).toBe(false)
+    expect(slackChannelName('^')).toBe(false)
+    expect(slackChannelName('&')).toBe(false)
+    expect(slackChannelName('*')).toBe(false)
+    expect(slackChannelName('?')).toBe(false)
+    expect(slackChannelName('(')).toBe(false)
+    expect(slackChannelName(')')).toBe(false)
+    expect(slackChannelName('{')).toBe(false)
+    expect(slackChannelName('}')).toBe(false)
+    expect(slackChannelName('<')).toBe(false)
+    expect(slackChannelName('>')).toBe(false)
+    expect(slackChannelName('=')).toBe(false)
+    expect(slackChannelName('+')).toBe(false)
+    expect(slackChannelName('|')).toBe(false)
+    expect(slackChannelName('・')).toBe(false)
+    // eslint-disable-next-line @typescript-eslint/quotes
+    expect(slackChannelName("'")).toBe(false)
+    expect(slackChannelName('`')).toBe(false)
+    expect(slackChannelName('~')).toBe(false)
+    expect(slackChannelName('/')).toBe(false)
+    // Single back slash can't be tested because it's an escape character
+    expect(slackChannelName('\\')).toBe(false)
```

Comment:
Just in case, I am right, ain't I?

Reply:
That comment seems off because `'\\'` is "single" back slash. (`'\\'.length === 1`)

Reply:
I meant if I write like `'\'`, then 2nd quote is escaped and the string can't be closed.

Reply:
Yeah that won’t work 😅

Reply:
Ah, sorry I misunderstood your comment.
You are right.

Reply:
Fixed: 7eb6c05